Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 let package = Package(
     name: "reminders",
     platforms: [
-        .macOS(.v10_15)
+        .macOS(.v12)
```

Comment:
Required so that we can use `ISO8601FormatStyle` and avoid:

```
error: 'ISO8601FormatStyle' is only available in macOS 12.0 or newer
```

Reply:
eh, fine with me